data/sql_updates/communication_cost.sql

```diff
@@ -3,26 +3,30 @@ create materialized view ofec_communication_cost_mv_tmp as
 select
     row_number() over () as idx,
     *,
+    s_o_cand_id as cand_id,
+    org_id as cmte_id,
     report_pdf_url(image_num) as pdf_url
-from communication_costs_vw
-where rpt_yr >= :START_YEAR
+from nml_form_76
```

Comment:
We actually want `fec_vsum_f76`. I gave you the wrong instructions- I just found out  `nml_form_76` is also a data warehouse table.